cedar-policy-validator/protobuf_schema/Validator.proto

```diff
@@ -102,8 +102,8 @@ message EntityRecordKind {
 }
 
 enum OpenTag {
-        OpenAttributes = 0;
-        ClosedAttributes = 1;
+    OpenAttributes = 0;
```

Comment:
A general note about the protobuf impl. Does this mean that `OpenAttributes` on records in the schema would become a publicly accessible feature when we stabilize protobuf? What other internal details might be exposed this way?

Reply:
Interesting question.

Protobuf is not self-describing.  In this PR, public-API users only see a blob of bytes, and if one of those bits indicates open-attributes vs closed-attributes, they would never know.  But if anyone actually relies on our `.proto` files to encode/decode themselves, then yes, they see these attributes.

I believe the Lean protobuf parser errors if it sees OpenAttributes.

Reply:
Arguably if we want to stabilize protobufs before OpenAttributes, we should remove OpenAttributes from the protobuf format.

Reply:
Yeah, I'd agree with that. And also

* [attributes on actions](https://github.com/cedar-policy/cedar/blob/770b9f0bff6cf3e4c0074c3153b9983adf598e78/cedar-policy-validator/protobuf_schema/Validator.proto#L53)
* [never](https://github.com/cedar-policy/cedar/blob/770b9f0bff6cf3e4c0074c3153b9983adf598e78/cedar-policy-validator/protobuf_schema/Validator.proto#L70) and [empty set](https://github.com/cedar-policy/cedar/blob/770b9f0bff6cf3e4c0074c3153b9983adf598e78/cedar-policy-validator/protobuf_schema/Validator.proto#L73) types
* [Action Entity](https://github.com/cedar-policy/cedar/blob/770b9f0bff6cf3e4c0074c3153b9983adf598e78/cedar-policy-validator/protobuf_schema/Validator.proto#L85) type
* and [permissive validation mode](https://github.com/cedar-policy/cedar/blob/770b9f0bff6cf3e4c0074c3153b9983adf598e78/cedar-policy-validator/protobuf_schema/Validator.proto#L124) (behind it's own feature)